homeassistant/components/hassio/__init__.py

```diff
@@ -610,7 +610,7 @@ async def _async_setup_hardware_integration(hass):
             )
         )
 
-    await _async_setup_hardware_integration(hass)
+    await _async_setup_hardware_integration(datetime.now())
```

Comment:
This argument is not used, it should be changed to `_: datetime | None = None` instead of passing this unused object.

Reply:
Should be ok in #92820